src/components/routes/Home.js

```diff
@@ -0,0 +1,60 @@
+import React, { useEffect, useState } from 'react';
+import HomeIcon from '@mui/icons-material/Home';
+import SettingsIcon from '@mui/icons-material/Settings';
+import { useSelector, useDispatch } from 'react-redux';
+import getCountries from 'features/FetchAPI';
+import CountriesList from '../CountriesList';
+
+const Homepage = () => {
+  const dispatch = useDispatch();
+  const countries = useSelector((state) => state.countries.countries);
+
+  const [searchItem, setSearchItem] = useState('');
+
+  useEffect(() => {
+    if (countries.length === 0) {
+      dispatch(getCountries());
+    }
+  }, [dispatch, countries.length]);
+
+  const newCountries = countries.filter(
+    (country) => country.name.toLowerCase().includes(searchItem.toLowerCase())
+      || country.region.toLowerCase().includes(searchItem.toLowerCase()),
+  );
+
+  const handleSearch = (e) => {
+    e.preventDefault();
+    setSearchItem(e.target.value);
+  };
+
+  return (
+    <div>
+      <header className="home-header">
+        <div className="logo">
+          <HomeIcon className="svg_icons" />
+        </div>
+        <div className="search">
+          {' '}
+          <input
+            type="text"
+            name="searchItem"
+            placeholder="search for a country"
+            value={searchItem}
+            onChange={handleSearch}
+          />
```

Comment:
**👔 Excellent implementation in response to the previous reviewer's request.**

- [ ] 👨‍💻 However, as previously mentioned by the reviewer, it is necessary to use a filter for loading data. While it is noted that you have implemented filtering for searching, it is also important to filter the main fetched data for loading purposes.

🙏 We kindly request that you take into account the previous reviewer's feedback and make the necessary adjustments. Thank you!

Reply:
Hello @Rustamxon7 can you explain to me further? I don't get the changes required

Reply:
Hello can i get a feedback atleast